### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MiKPo4eLiK**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
```
Comment (vsmutok) [inline app/main.py:20]: write your code in one line

**Review by vsmutok — COMMENTED** [review]
Why you close your PR?

File: app/main.py (hunk 1 above)
Comment (MiKPo4eLiK) [reply app/main.py:20]: Ok

**Comment by MiKPo4eLiK** [comment]
> Why you close your PR?

 I`m little confused

**Pull request closed without merging**